### PR TITLE
update PFJob struct

### PR DESCRIPTION
**Pull request by D0m021ng**

### CODE REVIEW

File: pkg/job/api/job.go [hunk 1]
```diff
@@ -111,25 +116,35 @@ type PFJob struct {
...
-		ClusterID: ClusterID(conf.GetClusterID()),
-		QueueID:   QueueID(conf.GetQueueID()),
-		FSID:      conf.GetFS(),
-		UserName:  conf.GetUserName(),
-		Conf:      *jobConf,
+		ID:        job.ID,
+		Name:      job.Name,
+		Namespace: job.Config.GetNamespace(),
+		JobType:   job.Config.Type(),
+		JobMode:   job.Config.GetJobMode(),
```
Comment (qiaoshuangshuang) [inline pkg/job/api/job.go:129]: job mode？

File: pkg/job/runtime/kubernetes/executor/paddlejob.go [hunk 2]
```diff
@@ -36,201 +36,171 @@ type PaddleJob struct {
 	JobModeParams
 }
 
-func (paddleJob *PaddleJob) validateJob() error {
-	if err := paddleJob.KubeJob.validateJob(); err != nil {
+func (pj *PaddleJob) validateJob() error {
+	if err := pj.KubeJob.validateJob(); err != nil {
 		return err
 	}
-	if len(paddleJob.JobMode) == 0 {
```
Comment (qiaoshuangshuang) [inline pkg/job/runtime/kubernetes/executor/paddlejob.go:43]: use framework and jobtype can guess the job mode？

Comment (D0m021ng) [reply pkg/job/runtime/kubernetes/executor/paddlejob.go:43]: ok

File: pkg/job/api/job.go (hunk 1 above)
Comment (D0m021ng) [reply pkg/job/api/job.go:129]: VCJob and SparkApp need `JobMode` to identify distributed job mode, such as `NCCL` or `PServer`.

**Review by qiaoshuangshuang — APPROVED** [review]

**Pull request merged**